test_harness/Cargo.toml

```
@@ -29,3 +29,4 @@ tokio = { version = "1", features = ["rt-multi-thread", "macros", "time"] }
 # probably needed for Relayer calls
 # wiremock = "0.5"
 pretty_assertions = "^1.0"
+serde_qs = "0.9.2"
```

```suggestion
serde_qs = "^0.9"
```

You would also need to update cargo.lock though